pkg/controllers/selection/preferences.go

```diff
@@ -71,6 +71,8 @@ func (p *Preferences) Relax(ctx context.Context, pod *v1.Pod) {
 
 func (p *Preferences) relax(ctx context.Context, pod *v1.Pod) bool {
 	for _, relaxFunc := range []func(*v1.Pod) *string{
+		func(pod *v1.Pod) *string { return p.removePreferredPodAffinityTerm(pod) },
+		func(pod *v1.Pod) *string { return p.removePreferredPodAntiAffinityTerm(pod) },
```

Comment:
I'm not sure about the positioning of these 2 relaxation functions.
Should they go after `removePreferredNodeAffinityTerm`?

Reply:
I think this is fine since especially for now they will always need to be removed.